Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -37,6 +37,11 @@
 import octobot_trading.modes.script_keywords as script_keywords
 
 
+class DCAActivity(enum.Enum):
+    CREATED_ORDERS = "created_orders"
+    NOTHING_TO_DO = "nothing_to_do"
```

Comment:
Should we use an common enum or it's better to create dedicated keys for DCA ?

Reply:
we also need to be able to add details on what happend (details on what go rebalanced in indexes for example). I'll change this